daemon/container_operations_unix.go

```diff
@@ -760,10 +764,6 @@ func (daemon *Daemon) connectToNetwork(container *container.Container, idOrName
 	controller := daemon.netController
 
 	if endpointConfig != nil {
```

Comment:
thats correct. but we must fail if the alias is used for default network (similar to `--ip`). Hence we must move it inside `updateNetworkConfig` & also remove the `isUserDefined()` check in `Parse()` for the Aliases case.